storage/store/approvals.go

```diff
@@ -1,35 +1,33 @@
-package badger
+package store
```

Comment:
GitHub correctly recognized the changes are actually on top of a file renaming, much easier to view the actual changes. And there is no `pebble` version to review, because this is a universal one.